examples/project_atproto_dashboard/dbt_project/profiles.yml

```
@@ -17,7 +17,7 @@ bluesky:
       type: duckdb
       schema: bluesky
       path: "md:prod_bluesky?MOTHERDUCK_TOKEN={{ env_var('MOTHERDUCK_TOKEN') }}"
```

We can also change this here per Jacob's suggestion
```suggestion
      path: "md:prod_bluesky"
```